admin-manual/installation-setup/installation/install-ansible.rst

```diff
@@ -65,17 +65,6 @@ Instructions
 
 3. Download the Ansible roles:
 
-   .. _ubuntu-14.04:
-
-   Ubuntu 14.04 (Trusty):
-
-   .. code:: bash
-
-      cd deploy-pub/playbooks/archivematica-trusty
-      ansible-galaxy install -f -p roles/ -r requirements.yml
-
-   .. _ubuntu-16.04:
```

Comment:
Is it ok to remove this?

Reply:
Thanks, I deleted this line by mistake. Just restored.